README.org

```diff
@@ -6,97 +6,141 @@ A sensible starting point for hacking your own Emacs configuration.
+from a =rational-*= module into their own configuration /without/
+using Rational Emacs.
 
 ** Prioritize built-in Emacs functionality
 
-Where possible, we will leverage built-in Emacs functionality instead of external packages, for example:
+Where possible, we will leverage built-in Emacs functionality instead
+of external packages, for example:
 
-- project.el instead of Projectile
-- =tab-bar-mode= instead of Perspective.el, =persp-mode=, eyebrowse, etc
-- eglot instead of lsp-mode (because eglot prioritizes built-in functionality)
+- =project.el= instead of =Projectile=
+- =tab-bar-mode= instead of =Perspective.el=, =persp-mode=,
+  =eyebrowse=, etc
+- =eglot= instead of =lsp-mode= (because =eglot= prioritizes built-in
+  functionality)
 - *Possibly* =vc-mode= by default
 
 ** Works well in the terminal
 
-Some people prefer to use Emacs in the terminal instead of as a graphical program.  This configuration should work well in this case too!  This also enables the use of Emacs in Termux on Android.
+Some people prefer to use Emacs in the terminal instead of as a
+graphical program. This configuration should work well in this case
+too! This also enables the use of Emacs in Termux on Android.
 
 ** Can be integrated with a Guix configuration
 
-It should be possible to customize aspects of the Rational Emacs configuration inside of a Guix Home configuration so that things like font sizes, themes, etc can be system-specific.
+It should be possible to customize aspects of the Rational Emacs
+configuration inside of a Guix Home configuration so that things like
+font sizes, themes, etc can be system-specific.
 
-It can also use packages installed via the Guix package manager instead of =straight.el=.
+It can also use packages installed via the Guix package manager
+instead of =straight.el=.
 
-** Works well with Chemacs2
+** Works well with =Chemacs2=
 
-Chemacs2 is an excellent tool for enabling the use of multiple Emacs configurations simultaneously.  This configuration will behave well when used with Chemacs2 so that users can try and use different Emacs configurations as needed.
+=Chemacs2= is an excellent tool for enabling the use of multiple Emacs
+configurations simultaneously. This configuration will behave well
+when used with =Chemacs2= so that users can try and use different
+Emacs configurations as needed.
 
 ** Helps you learn Emacs Lisp
 
-Instead of providing a higher-level configuration system out of the box like other Emacs configurations, we follow standard Emacs Lisp patterns so that you can learn by reading the configuration.
+Instead of providing a higher-level configuration system out of the
+box like other Emacs configurations, we follow standard Emacs Lisp
+patterns so that you can learn by reading the configuration.
 
 * Why use it?
 
-Why choose this configuration over Doom Emacs, Spacemacs, Prelude, or others?
+Why choose this configuration over =Doom Emacs=, =Spacemacs=,
```

Comment:
Since these are names of projects instead of code symbols, you might use italics instead with `/Doom Emacs/`, etc.

Reply:
Check!